cabal.project

```diff
@@ -11,15 +11,18 @@ repository cardano-haskell-packages
     c00aae8461a256275598500ea0e187588c35a5d5d7454fb57eac18d9edb86a56
     d4a35cd3121aa00d18544bb0ac01c3e1691d618f462c46129271bccf39f7e8ee
 
--- See ouroboros-network/CONTRIBUTING for some Nix commands you will need 
+-- See ouroboros-network/CONTRIBUTING for some Nix commands you will need
 -- to run if you update either of these.
 -- Bump this if you need newer packages from Hackage
-index-state: 2022-10-31T00:00:00Z
--- Bump this if you need newer packages from CHaP
-index-state: cardano-haskell-packages 2022-11-11T00:00:00Z
+index-state:
+  , hackage.haskell.org 2023-01-30T23:55:53Z
+  , cardano-haskell-packages 2023-02-14T10:24:01Z
 
 packages: ./.
 
+allow-newer:
+    monoidal-synchronisation:base
```

Comment:
This is potentially a problem. Is there an upstream issue?

Reply:
It's in ouroboros-network, so should they not have released a compatible one already?

Reply:
Yes, there is an upstream issue.

Reply:
Yes, they should have fixed this.

Reply:
It's fixed now: https://github.com/input-output-hk/cardano-haskell-packages/pull/132

Reply:
Upstream fixed, CHaP `index-state` updated, `allow-newer` removed.